src/patternfly/components/Nav/nav.scss

```diff
@@ -455,69 +257,47 @@
       left: 0;
       display: block;
       width: 100%;
-      height: var(--pf-c-nav__horizontal-list-link--after--Height);
+      height: var(--pf-c-nav__list-link--after--Height);
       content: "";
-      background-color: var(--pf-c-nav__horizontal-list-link--after--BackgroundColor);
     }
   }
 }
 
 // List tertiary
 .pf-c-nav__tertiary-list {
   .pf-c-nav__item {
-    margin-right: var(--pf-c-nav__tertiary-list-item--MarginRight);
+    margin-right: var(--pf-c-nav__horizontal-list-item--MarginRight);
   }
 
   // List tertiary link
   .pf-c-nav__link {
-    padding-top: var(--pf-c-nav__tertiary-list-link--PaddingTop);
-    padding-bottom: var(--pf-c-nav__tertiary-list-link--PaddingBottom);
-    font-weight: var(--pf-c-nav__tertiary-list-link--FontWeight);
-    color: var(--pf-c-nav__tertiary-list-link--Color);
+    padding-top: var(--pf-c-nav__list-link--PaddingY);
```

Comment:
looks like the original padding-top and padding-bottom were different, so these shouldn't have a shared var.

Reply:
Padding Top/Bottom is the same per design

<img width="528" alt="screen shot 2018-12-11 at 1 39 44 pm" src="https://user-images.githubusercontent.com/20118816/49822138-3f86b480-fd4a-11e8-8e70-90be764c3f12.png">

Reply:
It looks like it is, but the underline is about half way in the middle of the bottom padding in the design, and the underline is at the bottom of the padding per the CSS? 
<img width="398" alt="screen shot 2018-12-11 at 1 20 55 pm" src="https://user-images.githubusercontent.com/35148959/49824471-c8e8b780-fd47-11e8-8bd4-f9d5e7d7bfd0.png">

Reply:
fixed this